app/scripts/directives/realtimestring.js

```diff
@@ -167,20 +163,9 @@ angular.module('oncokbApp')
                     if (_.isUndefined($scope.tumorForms)) {
                         $scope.tumorForms = mainUtils.getTumorFormsByCancerTypes($scope.tumor.cancerTypes);
                     }
-                    if ($scope.pureContent.text === '1' || $scope.pureContent.text === '2A' ||
+                    if ($scope.pureContent.text === '1' || $scope.pureContent.text === '2' ||
                         ($rootScope.reviewMode && ($scope.data[$scope.key + '_review'].lastReviewed === '1' ||
-                            $scope.data[$scope.key + '_review'].lastReviewed === '2A'))) {
-                        _propagationOpts = [
-                            $scope.propagationOpts.no,
-                            $scope.propagationOpts['2B'],
-                            $scope.propagationOpts['4']
-                        ];
-                        if (propagationType === 'solid' && !$scope.tumorForms.includes('LIQUID')) {
-                            _propagation = setDefaultPropagation(propagationKey, '2B', initial)
-                        } else {
-                            _propagation = setDefaultPropagation(propagationKey, 'no', initial);
-                        }
-                    } else if ($scope.pureContent.text === '3A' ||
```

Comment:
How come this whole logic is removed?

Reply:
I merged 2 conditions together because their propagation options are the same, no, 3B and 4.